<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListView" name="listViewStreet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>25</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Фамилия Имя Отчество</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Поиск</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>H:/RIP/iCO/ICO/Search.ico</normaloff>H:/RIP/iCO/ICO/Search.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableView" name="tableViewMain">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
          <widget class="QTabWidget" name="tabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Начисления</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QTableView" name="tableViewAccruals">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Платежи</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QTableView" name="tableViewPayment"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_account">
    <property name="title">
     <string>Абонент</string>
    </property>
    <addaction name="action_add_account"/>
    <addaction name="separator"/>
    <addaction name="action_list_account"/>
   </widget>
   <widget class="QMenu" name="menu_rate">
    <property name="title">
     <string>Тариф</string>
    </property>
    <addaction name="action_add_rate"/>
    <addaction name="separator"/>
    <addaction name="action_list_rate"/>
   </widget>
   <widget class="QMenu" name="menu_street">
    <property name="title">
     <string>Улица</string>
    </property>
    <addaction name="action_add_street"/>
    <addaction name="separator"/>
    <addaction name="action_list_street"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Здание</string>
    </property>
    <addaction name="action_add_building"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_street"/>
   <addaction name="menu_account"/>
   <addaction name="menu_rate"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAccount"/>
   <addaction name="actionBuilding"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAccount">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/Add-Account.ico</normaloff>:/Resources/Add-Account.ico</iconset>
   </property>
   <property name="text">
    <string>Дабавить Абонента</string>
   </property>
   <property name="toolTip">
    <string>Дабавить Абонента</string>
   </property>
  </action>
  <action name="actionBuilding">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/building_add.ico</normaloff>:/Resources/building_add.ico</iconset>
   </property>
   <property name="text">
    <string>Добавить Здание</string>
   </property>
   <property name="toolTip">
    <string>Добавть новое здание</string>
   </property>
  </action>
  <action name="action_list_rate">
   <property name="text">
    <string>Список</string>
   </property>
  </action>
  <action name="action_add_rate">
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="iconText">
    <string>Добавить тариф</string>
   </property>
   <property name="toolTip">
    <string>Добавить тариф</string>
   </property>
  </action>
  <action name="action_add_account">
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Добавить акаунт</string>
   </property>
  </action>
  <action name="action_list_account">
   <property name="text">
    <string>Список</string>
   </property>
  </action>
  <action name="action_add_street">
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Добавить улицу</string>
   </property>
  </action>
  <action name="action_list_street">
   <property name="text">
    <string>Список</string>
   </property>
  </action>
  <action name="action_add_building">
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Добавить здание</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
